ansible/roles/common/tasks/main.yml

```diff
@@ -70,7 +70,7 @@
   changed_when: false
 
 - name: ensure service user {{user}} exists
-  user: name={{user}} system=yes createhome=no home=/nonexistent
+  user: name={{user}} system=yes createhome=yes home=/home/{{user}}
         shell=/usr/sbin/nologin
   become: true
-  when: id_user|failed
+  when: id_user.rc != 0
```

Comment:
This is as suggested by the Ansible docs, although #218 suggests `is failed` which is nicer to read? `|failed` filter definitely does not work any longer.